_includes/header.html

```diff
@@ -14,6 +14,9 @@
         </li>
       {% endif %}
     {% endfor %}
+    <li class="nav-item">
+      <a href="https://euruko2017.org/2017/06/21/diversity-program.html">Diversity</a>
```

Comment:
`/whatever/xy` is an absolute path in this case i think.
https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths

i think in general the [self] domain should not be used in links because it breaks testing locally.

thanks for the fix!